java/gradle.java/src/org/netbeans/modules/gradle/java/newproject/Wizards.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.gradle.java.newproject;
+
+import java.util.List;
+import org.netbeans.api.templates.TemplateRegistration;
+import org.netbeans.modules.gradle.spi.newproject.BaseGradleWizardIterator;
+import org.netbeans.modules.gradle.spi.newproject.GradleInitWizard;
+import org.netbeans.modules.gradle.spi.newproject.GradleInitWizard.TestFramework;
+import org.openide.util.NbBundle;
+
+import static org.netbeans.modules.gradle.spi.newproject.GradleInitWizard.TestFramework.*;
+
+/**
+ *
+ * @author lkishalmi
+ */
+public final class Wizards {
+
+    private Wizards() {};
+
+    private static final List<Integer> JAVA_VERSIONS = List.of(22, 21, 17, 11, 8);
+    private static final List<TestFramework> JAVA_TEST_FRAMEWORKS = List.of(
+            JUNIT,
+            JUNIT_5,
+            TESTNG
+    );
```

Comment:
this determines the order of the combo box?

consider to swap JUnit 4 and 5 so that the default option has a "latest bias"

Reply:
Added a line to prefer Java 5 framework.